packages/aws-cdk-lib/aws-ec2/test/vpc-endpoint.test.ts

```diff
@@ -694,6 +694,33 @@ describe('vpc endpoint', () => {
       });
 
     });
+
+    test.each([
+      ['us-isof-test-1', 'gov.ic.hci.csp'],
+      ['eu-isoe-test-1', 'uk.adc-e.cloud'],
+    ])('test vpc interface endpoint for ECR can be created correctly in isolated regions', (region : string, domain: string) => {
```

Comment:
can we add tests for commercial regions, cn regions, govcloud regions?

Reply:
sure, though we already had tests for cn regions below this, added it here too.